test/pyqrack/runtime/noise/native/test_pauli.py

```diff
@@ -126,5 +144,6 @@ def test_atom_loss():
 
 if __name__ == "__main__":
     test_pauli_channel()
+    test_pauli_probs_check()
     test_cz_pauli_channel_false()
     test_cz_pauli_channel_true()
```

Comment:
Oh we should probably get rid of this going forward. this is pet peeve of @Roger-luo

Reply:
Great, I very much agree with that :smile:

Reply:
do you know you can do

```
pytest test/pyqrack/runtime/noise/native/test_pauli.py -k test_pauli_probs_check
```

to trigger a specific test?

Reply:
Yes. Still find it more convenient to hit the run script button in vs code though 😄